api-reference/docstubs/api/calendar-delete-calendarview.md

```diff
@@ -0,0 +1,67 @@
+---
+title: "Delete calendarView"
+description: "Delete a calendarView object."
+author: "**TODO: Provide Github Name**"
+localization_priority: Normal
+ms.prod: "**TODO: Add MS prod**"
+doc_type: apiPageType
+---
+
+# Delete calendarView
+
+Namespace: microsoft.graph
+
+Delete a calendarView object.
+
+## Permissions
+One of the following permissions is required to call this API. To learn more, including how to choose permissions, see [Permissions](/concepts/permissions-reference.md).
+
+|Permission type|Permissions (from most to least privileged)|
+|:---|:---|
+|Delegated (work or school account)|**TODO: Provide applicable permissions.**|
+|Delegated (personal Microsoft account)|**TODO: Provide applicable permissions.**|
+|Application|**TODO: Provide applicable permissions.**|
+
+## HTTP request
+<!-- {
+  "blockType": "ignored"
+}
+-->
+``` http
+DELETE /invitations/{invitationsId}/invitedUser/messages/{messageId}/event/calendar/calendarView
```

Comment:
I don't think the Ruby tool generates calendar-delete-calendarview.md. 
Also, line 31 doesn't look like an intended use case.
Are we assuming authors will weed out topics like this before publishing?